src/stdlib/encode_json.rs

```diff
@@ -1,8 +1,15 @@
 use crate::compiler::prelude::*;
 
-fn encode_json(value: Value) -> Resolved {
+fn encode_json(value: Value, pretty: bool) -> Resolved {
```

Comment:
It is a good practice to avoid `bool` parameters. There are many ways to refactor: 

- an options `struct`
- `enum` instead of `bool`
- extract implementation details and split `encode_json` into two functions